kurl_util/cmd/yamltobash/main.go

```diff
@@ -16,7 +16,7 @@ import (
 	kurlscheme "github.com/replicatedhq/kurl/kurlkinds/client/kurlclientset/scheme"
 	kurlv1beta1 "github.com/replicatedhq/kurl/kurlkinds/pkg/apis/cluster/v1beta1"
 	kurlversion "github.com/replicatedhq/kurl/pkg/version"
-	"gopkg.in/yaml.v1"
```

Comment:
@areed is there any reason this is using v1?

Reply:
Not that I'm aware of